modules/apps/feature-flag/feature-flag-web/src/main/resources/META-INF/resources/js/FeatureFlagToggle.tsx

```diff
@@ -47,7 +49,12 @@ const FeatureFlagToggle = ({
 			);
 
 			if (response.ok) {
+				const res = await response.json();
```

Comment:
`data` or `jsonData` are better names for the variable here.

Reply:
Changed to `responseData`